README.md

```diff
@@ -565,6 +565,9 @@ $ krane render --bindings='@config/production.json'
 
 # Load YAML file from ./config (.yaml or yml supported)
 $ krane render --bindings='@config/production.yaml'
+
+# Load multiple files via a space separated string
+$ krane render --bindings='@config/production.yaml' '@config/common.yaml'
```

Comment:
:+1: This looks ok to me. As @dturn mentioned we'll be adding back the ability to specify certain flags multiple times but that is blocked on an upstream PR in [thor](https://github.com/erikhuda/thor/pull/713). Once that merges I might remove this in favour of the repeatable method, but that's a consideration for the future.